docs/examples/downloading_files_r0.md

```diff
@@ -0,0 +1,99 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+# Downloading Data Based on Seeing
+
+```{code-cell} python
+import matplotlib.pyplot as plt
+from sunpy.net import Fido, attrs as a
+
+import dkist
+import dkist.net
+```
+
+Let's find a dataset with the highest average value of r0 (this is bad?).
+First we'll search for all unembargoed VISP data, as embargoed data is no use to us for this exercise.
+
+```{code-cell} python
+res = Fido.search(a.Instrument("VISP"), a.dkist.Embargoed(False))
+```
+
+Next, since we want to use the highest average $r_0$, we can have Fido sort the results and output just the useful columns.
+
+```{code-cell} python
+res['dkist'].sort("Average Fried Parameter", reverse=True)
+res['dkist'].show("Dataset ID", "Start Time", "Average Fried Parameter", "Primary Proposal ID")
+```
+
+We'll ignore the datasets with anomalously high values and look at the next one with a value that is high but realistic.
+This gives us the dataset `AXVXL`.
+We can download the dataset ASDF with Fido to inspect it in more detail.
+Remember that this only downloads a single ASDF file with some more metadata about the dataset, not the actual science data.
+
+```{code-cell} python
+:tags: [remove-stderr]
+asdf_files = Fido.fetch(res['dkist'][3], path="~/sunpy/data/{dataset_id}/")
+ds = dkist.load_dataset(asdf_files[0])
+```
+
+Now that we have access to the FITS headers we can inspect the $r_0$ more closely, just as we did in the previous notebook.
+Remember that `DINDEX4` is the Stokes index, so we can plot the $r_0$ for just Stokes I like so:
+```{code-cell} python
+plt.plot(ds.headers[ds.headers["DINDEX4"] == 1]["ATMOS_R0"])
+```
+
+Now let's slice down our dataset based on the first frame where $r_0$ is high:
+
+```{code-cell} python
+# Select headers for only frames with bad r0
+bad_headers = ds.headers[ds.headers["ATMOS_R0"] > 1]
+
+# Make sure headers are sorted by date
+bad_headers.sort("DATE-AVG")
+
+# Slice up to the index of the first bad frame
+sds = ds[0, :bad_headers[0]["DINDEX3"]-1, :, :]
+```
+
+We can now download only these files, remember you need globus-connect-personal running for this.
```

Comment:
We're going to need to caveat this for the docs or something as these plots wont exist.